ImageGallery/src/org/sleuthkit/autopsy/imagegallery/actions/OpenAction.java

```diff
@@ -285,10 +286,9 @@ private void openTopComponent() {
         SwingUtilities.invokeLater(() -> {
             try {
                 ImageGalleryTopComponent.openTopComponent();
-            } catch (NoCurrentCaseException ex) {
-                logger.log(Level.SEVERE, "Attempted to access ImageGallery with no case open.", ex);//NON-NLS
             } catch (TskCoreException ex) {
-                logger.log(Level.SEVERE, "Error getting ImageGalleryController.", ex); //NON-NLS}
+                logger.log(Level.SEVERE, "Failed to open Image Gallery top component", ex); //NON-NLS}
```

Comment:
My reasoning here is that the user just pushed a toolbar button or selected a menu item and nothing happened. So I want to give him or her some feedback that we tried to open the Image Gallery, but failed. Otherwise, the user might push the button multiple times in frustration.

Reply:
yes that is what I thought,